packages/ionic-angular/src/components/tabs/tab.ts

```diff
@@ -307,7 +307,7 @@ export class Tab extends NavControllerBase implements ITab {
       this._dom.read(() => {
         this.resize();
       });
```

Comment:
This code is divergent from what is in master in that master has some logic that is not included here (before my change). Not sure if that is an issue or not.